### ISSUE TITLE
Make the theme accessible according to WCAG

### ISSUE THREAD

**Issue by Reedyn** [Reedyn/Saga#120]
* [x] Remake icons in menu (needs to have a descriptor)
* [x] H1 for screen readers on index page (add `.sr-only` helper class)
* [x] Remove nested links in feed.
* [x] Identifiers for navigation
* [x] Color contrast everywhere
* [x] Option to skip to main content
* [x] "Read more" link in post should be included in landmarks
